### PR TITLE
Adding github actions removing travis

### PR DESCRIPTION
**Pull request by ydovzhenko**

### CODE REVIEW

File: .github/workflows/tests_and_lint.yml [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+    branches:
+      - "*"
+jobs:
+  lint:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: [3.8]
+    steps:
+      - name: Install Python dependencies
```
Comment (poneill) [inline .github/workflows/tests_and_lint.yml:19]: nit: for debugging later on,

`      - name: Set up Python ${{ matrix.python-version }}`

File: .github/workflows/tests_and_lint.yml [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+        run: |
+          pipenv install --dev --deploy --system
```
Comment (poneill) [inline .github/workflows/tests_and_lint.yml:21]: I get `pipenv not found` errors when I try to run this locally using `act`.

File: .github/workflows/tests_and_lint.yml (hunk 1 above)
Comment (poneill) [reply .github/workflows/tests_and_lint.yml:19]: you might need to add something of the form:

```      - name: Install pipenv
        run: |
          python -m pip install --upgrade pipenv wheel
```

under steps, see e.g.: https://gist.github.com/alukach/6f3a371e9af600e417aca1b36806ad72

**Review by obi1kenobi — COMMENTED** [review]
You probably didn't mean to check in the `.idea` directory here, right? Local editor config probably shouldn't be in the repo.

**Comment by ydovzhenko** [comment]
@obi1kenobi yup, still working on this. I'll ping you when it's all fixed up (and I've removed the .idea dir :-)) 
Right now, when I run `act` locally, getting a few of this type of error during linting: `bubs/embedding_layer.py:6:0: E0611: No name 'layers' in module 'LazyLoader' (no-name-in-module)`

**Review by poneill — APPROVED** [review]
:crossed_fingers:

**Pull request merged**